operator/utils/utils.go

```diff
@@ -9,6 +9,44 @@ import (
 	v1 "k8s.io/api/core/v1"
 )
 
+// Attempts to add environment variable but does NOT override if already existing
+func AddEnvVarToDeploymentContainers(deploy *appsv1.Deployment, envVar *v1.EnvVar) {
+	for containerIdx := 0; containerIdx < len(deploy.Spec.Template.Spec.Containers); containerIdx++ {
+		deployContainer := &deploy.Spec.Template.Spec.Containers[containerIdx]
+		// Before adding it ensures it doesn't exist as it will NOT override
+		for _, e := range deployContainer.Env {
+			if e.Name == envVar.Name {
+				break
+			}
+		}
+		deployContainer.Env = append(deployContainer.Env, *envVar)
+	}
+}
+
+// Create a volume from the secret provided and mounts it to all the containers of deployment
+func MountSecretToDeploymentContainers(deploy *appsv1.Deployment, secretRefName, containerMountPath string) {
+	volumeName := "cert-volume"
```

Comment:
nit: should we call it `seldon-cert-volume` to avoid potential clashes with other volumes? I'm thinking of the [issue we hit recently with Istio](https://github.com/istio/istio/issues/22246), where a volume named `podinfo` was clashing with Seldon's own `podinfo` volume.

Reply:
Yes definitely! That's a good point, in general we should name them, it should not be a magic string either